portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,24 +39,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     
+    // keeping this just in case I need a filter for future reference
     // Filter movieFilter = new FilterPredicate("movie", FilterOperator.EQUAL, movie);
     // Query query = new Query("Comment").setFilter(movieFilter).addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
It's better to take these out to keep your code clean.

Sometimes I save these in a notes google doc or on a Google Keep note.